Tests/BackendIntegrationTests/BaseStoreKitIntegrationTests.swift

```diff
@@ -291,6 +324,14 @@ extension BaseStoreKitIntegrationTests {
         return try XCTUnwrap(transactions.first)
     }
 
+    // This should be "Finishing transaction".
+    static let finishingTransactionLog: String = Strings.purchase
+        .finishing_transaction(MockStoreTransaction())
+        .description
+        .components(separatedBy: " ")
+        .prefix(2)
+        .joined(separator: " ")
+
```

Comment:
Welcome to feedback if anybody thinks this is overkill.

Reply:
lol yeah, kinda. I'd either just look for "Finishing transaction" or maybe update the string in the log so it's meaningful to us (like adding `action: finish_transaction`) to the log or something

Like, if you wanted to change that string it'd be easier to understand that a change like "will finish transaction" breaks a place where we're explicitly looking for "Finishing Transaction", than it is to understand why it broke this line